### DISCUSSION TITLE
What's the best way to merge clips?

### DISCUSSION THREAD

**Discussion by alexanderadam** [KyleKun/one_second_diary#89] [Q&A] [answered]
What would be the best way to merge clips manually until [feature 75](https://github.com/KyleKun/one_second_diary/issues/74) is implemented?

Should I use certain `ffmpeg` parameters to avoid quality loss or so?
Should I keep anything in mind regarding formats?

[These examples](https://stackoverflow.com/a/11175851) look plausible but I'm not sure which to choose.

**Comment by KyleKun** [comment, marked as answer]
Using `concat` filter is the way to go, it is lossless and it's the way we do it on the app:

1) Create File List
echo file video1.mp4 >  mylist.txt 
echo file video2.mp4 >> mylist.txt
echo file video3.mp4 >> mylist.txt

2) Run the ffmpeg command
ffmpeg -f concat -i mylist.txt -c copy DateProperlyParsed.mp4
